dashboard/app/controllers/helpful_links_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HelpfulLinksController < ApplicationController
+  def index
+    helpful_links_path
```

Comment:
Why do you need this?

Reply:
My greenness is showing. To be honest, I didn't have one until I kept getting this error no matter what I called the route, and adding this tiny controller made it all suddenly work. I was following along with the pools link, and it also has a very tiny controller. 
<img width="815" alt="Screen Shot 2022-10-26 at 4 16 27 PM" src="https://user-images.githubusercontent.com/37230822/198157625-e2097f67-ce9b-462a-a8b8-e64c0086b53f.png">

Reply:
It might work with an empty function. I do wonder if there's a way you can skip the controller all together but I'm not sure if that's possible/worth it

Reply:
Okay after some stackoverflow and tutorial research I've found that you could add something to a views/application file (which we don't have yet) for cases like this. Otherwise, it is a common and recommended convention to have an empty controller because the route will automatically look for the view supplied in the get call. https://stackoverflow.com/questions/8599404/rails-doesnt-need-index-method-in-controller-defined I've emptied out the controller and plan to leave it!